conf/machine/imx8mnul-ddr3l-evk.conf

```diff
@@ -0,0 +1,68 @@
+#@TYPE: Machine
+#@NAME: NXP i.MX 8MNUL EVK with DDR3L
+#@SOC: i.MX8MN
+#@DESCRIPTION: Machine configuration for NXP i.MX 8M Nano UltraLite Evaluation Kit with DDR3L
+#@MAINTAINER: Jun Zhu <junzhu@nxp.com>
+
+MACHINEOVERRIDES =. "imx-boot-container:mx8mnul:"
+
+require conf/machine/include/imx-base.inc
+require conf/machine/include/arm/armv8a/tune-cortexa53.inc
+
+MACHINE_FEATURES += " wifi bluetooth bcm43455 bcm4356"
+
+# NXP BSP can consume proprietary jailhouse and Broadcom drivers
+# OP-TEE is also applicable to NXP BSP, mainline BSP seems not to have
+# a full support for it yet.
+MACHINE_FEATURES:append:use-nxp-bsp = " optee jailhouse bcm4359"
+
+KERNEL_DEVICETREE_BASENAME = "imx8mn-ddr3l-evk"
```

Comment:
Seems wrong. This refers to `imx8mn`.

Reply:
Unfortunately, it is correct. I will make a comment to make it clear.

Reply:
How so? If this is the case why we have another machine and not choose this at runtime?

Reply:
All three SoCs in the 8M Nano family (Nano, Nano Lite, and Nano UltraLite) share the same SoC ID, so the same devicetree is compatible.